### PR TITLE
Add create dynamic badge job

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Add job in nightly-build(scala-build, docker-build-bigdl, python-build, python-sourceforge-build),nightly-test,license-scan, scala-style-check and python-style-check.

### CODE REVIEW

File: .github/actions/create-job-status-badge/action.yml [hunk 1]
```diff
@@ -15,6 +15,10 @@ inputs:
     required: true
   job-name:
     description: 'job name'
+  isSelfHostedRunner:
```
Comment (liu-shaojun) [inline .github/actions/create-job-status-badge/action.yml:18]: is-self-hosted-runner:

File: .github/actions/create-job-status-badge/action.yml [hunk 2]
```diff
@@ -38,6 +42,7 @@ runs:
       with:
         auth: ${{ inputs.secret }}
         gistID: ${{ inputs.gist-id }}
+        isSelfHostedRunner: ${{ inputs.isSelfHostedRunner }}
```
Comment (liu-shaojun) [inline .github/actions/create-job-status-badge/action.yml:45]: isSelfHostedRunner: ${{ inputs.is-self-hosted-runner }}

File: .github/actions/create-job-status-badge/action.yml [hunk 3]
```diff
@@ -49,6 +54,7 @@ runs:
...
```
Comment (liu-shaojun) [inline .github/actions/create-job-status-badge/action.yml:57]: isSelfHostedRunner: ${{ inputs.is-self-hosted-runner }}

**Review by liu-shaojun — APPROVED** [review]

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**